framework/rbac/DbManager.php

```diff
@@ -1008,18 +1008,59 @@ public function loadFromCache()
     /**
      * Returns all role assignment information for the specified role.
      * @param string $roleName
-     * @return string[] the ids. An empty array will be
+     * @param bool $recursively whether to search userids recursively.
+     * If it is true, all users assigned roleName role and roleName's all parent
+     * roles will be found.
+     * @return \string[] the ids. An empty array will be
      * returned if role is not assigned to any user.
      * @since 2.0.7
      */
-    public function getUserIdsByRole($roleName)
+    public function getUserIdsByRole($roleName, $recursively = false)
```

Comment:
Can't merge it into 2.0.x since method signature was changed. It breaks backwards compatibility.

Could be either merged into 2.1 or adjusted to keep method signature.